utils/sl/prereq_tools/base.py

```diff
@@ -284,7 +286,7 @@ def run_commands(self, commands, subdir=None):
                 retval = True
             else:
                 print('RUN: %s' % command)
-                if subprocess.call(command, shell=True,
```

Comment:
Update with # nosec
command.split(' ") and  removal of shell=True causing failure in multiple places:
Test stage Build on Leap 15 with Intel-C and TARGET_PREFIX completed with status FAILURE. https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-4130/10/execution/node/341/log

Reply:
I'll take a look at this Monday.

Reply:
This error was caused by an errant ' ' on the end of daos/utils/sl/components/init.py:287